docs/content/configuration/policy-resource.md

```diff
@@ -214,6 +214,42 @@ policies:
 ```
 In this example the Ingress Controller will use the configuration from the first policy reference `jwt-policy-one`, and ignores `jwt-policy-two`.
 
+### JWT using JWKS from remote location
+
+> Note: This feature is only available in NGINX Plus.
+
+The JWT policy configures NGINX Plus to authenticate client requests using JSON Web Tokens, this particular method allows import of the keys (JWKS) for JWT policy by means of a URL (for a remote server or an identity provider) so they don't have to be copied and updated to the IC pod.
```

Comment:
```suggestion
The JWT policy configures NGINX Plus to authenticate client requests using JSON Web Tokens, allowing import of the keys (JWKS) for JWT policy by means of a URL (For a remote server or an identity provider).
```

Reply:
More concise; the utility of the feature is self-evident and doesn't need to be explicitly stated.